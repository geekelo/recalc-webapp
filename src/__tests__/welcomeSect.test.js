import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import WelcomeSection from '../componenets/welcomeSection';

describe('snap shot for this component', () => {
  it('tests snapshot', () => {
    const tree = renderer.create(
      <BrowserRouter>
        <WelcomeSection />
      </BrowserRouter>
    ).JSON
    expect(tree).toMatchSnapshot();
  })

})