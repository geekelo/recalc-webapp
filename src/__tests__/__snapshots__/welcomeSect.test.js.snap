// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `undefined`;

exports[`snap shot for this component tests snapshot 1`] = `undefined`;
